patches/server/0968-New-event-verbose-logging-command.patch

```diff
@@ -0,0 +1,264 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: RyGuy <TheRealRyGuy@users.noreply.github.com>
+Date: Tue, 21 Mar 2023 16:21:07 -0400
+Subject: [PATCH] New event verbose logging command
+
+
+diff --git a/src/main/java/io/papermc/paper/command/PaperCommand.java b/src/main/java/io/papermc/paper/command/PaperCommand.java
+index 7ba60b4b4f29a42c58d98aafc5ea0fa3214f554c..4aaa903c390593507f46d2a5b25d581e446a791b 100644
+--- a/src/main/java/io/papermc/paper/command/PaperCommand.java
++++ b/src/main/java/io/papermc/paper/command/PaperCommand.java
+@@ -45,6 +45,7 @@ public final class PaperCommand extends Command {
+         commands.put(Set.of("dumpitem"), new DumpItemCommand());
+         commands.put(Set.of("mobcaps", "playermobcaps"), new MobcapsCommand());
+         commands.put(Set.of("dumplisteners"), new DumpListenersCommand());
++        commands.put(Set.of("dumpevents"), EventVerboseCommand.get());
```

Comment:
This dump event name is rather misleading, it's logging, capturing, or whatever.
I think a better name might be good, and also make the class name match the command.

Reply:
thoughts on `/logevent [event]` and `/logevents`?